services/checks/funcs.go

```diff
@@ -80,3 +82,52 @@ func formatVersionNum(args ...interface{}) (interface{}, error) {
 	}
 	return p.String(), nil
 }
+
+// GetAdditionalContext returns additional functions to be used in check scripts.
+func GetAdditionalContext() map[string]starlark.GoFunc {
+	return map[string]starlark.GoFunc{
+		"ip_is_private": ipIsPrivate,
```

Comment:
Let's add our two predeclared functions there too. That way all checks can use only context for everything